tests/translator/input/function_event_conditions.yaml

```diff
@@ -79,6 +79,22 @@ Resources:
             Topic:
               Ref: Notifications
 
+        SNSTopicWithSQSSubscription:
+          Type: SNS
+          Properties:
+            Topic:
+              Ref: Notifications
+            SqsSubscription:
+              QueueArn: !GetAtt Queue.Arn
+              QueueUrl: !Ref Queue
+
+        AnotherSNSWithSQSSubscription:
```

Comment:
I'd prefer a more descriptive name to immediately figure out what is the difference between this one and the one above, what each of them tests.